stdlib/2/email/message.pyi

```diff
@@ -0,0 +1,41 @@
+class Message:
+    preamble = ...
```

Comment:
Looks like `epilogue` is also a public attribute.

Reply:
Yep, missed it!